internal/browser/multiline/integrator.go

```diff
@@ -0,0 +1,75 @@
+package multiline
+
+import (
+	"strings"
+
+	"github.com/charmbracelet/bubbles/textinput"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+)
+
+type SubmitMsg string
+
+type model struct {
+	textInput     Model
+	err           error
+	ready         bool
+	keyboardFocus bool
+}
+
+func InitTextArea() *model {
+	ti := New()
+	ti.Placeholder = "sql query here"
+	ti.Focus()
+	return &model{
+		textInput:     ti,
+		err:           nil,
+		keyboardFocus: true,
+	}
+}
+
+func (m model) Init() tea.Cmd {
+	return textinput.Blink
+}
+
+func (m model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
+	switch tmsg := msg.(type) {
+	case tea.KeyMsg:
+		if tmsg.Type == tea.KeyTab {
+			var tmp tea.Cmd
+			if m.keyboardFocus {
+				m.textInput.Blur()
+			} else {
+				tmp = m.textInput.Focus()
+			}
+			m.keyboardFocus = !m.keyboardFocus
+			return m, tmp
+		}
+		if !m.keyboardFocus {
+			return m, nil
+		}
+		switch tmsg.Type {
+		case tea.KeyCtrlE:
+			return m, func() tea.Msg {
```

Comment:
Does it make sense to move the function to a variable and `return m, f`?

Reply:
Sure https://github.com/hazelcast/hazelcast-commandline-client/pull/65/commits/5e1e26717a9422c23bd7961a1db0037309d91af8